File: returnStr.py
def even_chars(st): 
    if len(st) > 100 or len(st) <2:
        return 'invalid string'
    arr = list(st)
    newArr = []
    count = 0
    for x in arr:
        count += 1
        if count % 2 == 0:
            newArr.append(x)
    return newArr

File: returnStr.js
function evenChars(string) {
    if (string.length > 100 || string.length <2){
      return 'invalid string'
    }
    let arr = string.split('')
    let newArr = []
    for (let i = 0; i <arr.length; i++){
      if (i % 2 != 0){
        newArr.push(arr[i])
      }
    }
      return newArr
    }